src/testHelpers/authUserInputs.js

```diff
@@ -22,6 +23,38 @@ export const createMockFirebaseUserAdminSDK = () => ({
   // ... other properties
 })
 
+/**
+ * same as createMockFirebaseUserAdminSDK but you can add claims!
+ * @param {*} claims {Obj} consisting of customClaims
+ * @returns {Obj} a decodedIdToken with extra claim properties
+ */
+export const createMockDecodedIdTokenWithClaims = (claims = {}) => ({
```

Comment:
Given this is the same as above, it seems unnecessary. Let's remove it, and in the the test that uses it, just spread the result:
```js
const idTokenWithClaims = { ...createMockFirebaseUserAdminSDK(), claims: { stuff: 'here' }}
```